package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "electron-unhandled",
-	"version": "3.0.2",
+	"version": "3.0.3",
```

Comment:
Never bump in a PR. That's up to the maintainers to do.

Reply:
That's an oops from working across too many projects at once.

Reply:
Do you want me to submit a PR to reverse this or is it trivial enough to resolve when you determine the appropriate version for these changes?